Dockerfile

```diff
@@ -3,6 +3,16 @@
 FROM ubuntu:16.04
 MAINTAINER Michael Stucki <michael@stucki.io>
 
+
+ENV \
+# ccache specifics
+    CCACHE_SIZE=50G \
+    CCACHE_DIR=/srv/ccache \
+    USE_CCACHE=1 \
+    CCACHE_COMPRESS=1 \
+# Extra include PATH, it may not include /usr/local/(s)bin on some systems
```

Comment:
Please set a 2nd `ENV` for this. I am not a fan of using comments after escaped linebreaks. I'm surprised that this works after all...